equinoxfitness/datacoco-cloud
# Upgrade Dependency Conflicts

## Description

Upgraded version of requests and boto3 in install_requires of setup.py

## Changes

- Upgraded `requests` to `2.25.1` in `install_requires` of `setup.py`
- Upgraded `boto3` to `1.16.48` in `install_requires` of `setup.py`
- Added item in `CHANGELOG.MD`


### How Has This Been Tested

1. Checkout this branch
2. Delete your existing `virtual environment` if you have
3. Create new clean `virtual environment`
4. Run the cli `pip3 install -r requirements.txt`
- [ ]   Verify that all depedencies have been successfully installed
